rosbag2_tests/test/rosbag2_tests/record_fixture.hpp

```diff
@@ -69,6 +69,23 @@ class RecordFixture : public TemporaryDirectoryFixture
     rclcpp::shutdown();
   }
 
+  void wait_for_metadata(int timeout_in_sec = 5)
+  {
+    const auto metadata_path = rcpputils::fs::path{root_bag_path_} / "metadata.yaml";
```

Comment:
After rebase, this should be:
```suggestion
    const auto metadata_path = root_bag_path_ / "metadata.yaml";
```